core/src/main/java/com/novoda/noplayer/NoPlayer.java

```diff
@@ -266,6 +267,19 @@
      */
     void setMaxVideoBitrate(int maxVideoBitrate);
 
+    /**
+     * Sets the maximum video size allowed. If the content contains tracks above the specified video size,
+     * those tracks will not be selected.
+     *
+     * @param maxVideoSize The maxium video size in pixels
+     */
+    void setMaxVideoSize(Dimension maxVideoSize);
+
+    /**
+     * Clears the maximum video size, if set.
+     */
+    void clearMaxVideoSize();
```

Comment:
For my knowledge: 
1- max size has to be set when initializing the player or it can be set anytime?
2- Can a max size be cleared while video in being played?

Reply:
Yes to both. You can set an initial max size when we create the player and you can change the max video size while the video is already playing. Those two scenarios are coded in the demo application ;)

Reply:
thanks. Fair enough.